application/Config/Paths.php

```diff
@@ -75,7 +75,7 @@ class Paths
 	 * can change this to `public_html`, for example, to comply
 	 * with your host's needs.
 	 */
-	public $publicDirectory = 'public';
+	public $publicDirectory = FCPATH;
```

Comment:
This property is not used in no place. I think it can be removed, since index.php and spark set the FCPATH.